samples/nrf5340/extxip_smp_svr/README.rst

```diff
@@ -43,6 +43,7 @@ Building and running
 
 .. |sample path| replace:: :file:`samples/nrf5340/extxip_smp_svr`
 .. include:: /includes/build_and_run.txt
+This sample need to by build with --no-sysbuild option.
```

Comment:
```suggestion
Because sysbuild is currently not supported, you must add the ``--no-sysbuild`` argument when building the sample:

.. parsed-literal::
   :class: highlight

   west build -b *board_target* --no-sysbuild
```
Would it be helpful to include an example command?

Reply:
I agree